modules/install/pages/upgrade.adoc

```diff
@@ -1,73 +1,94 @@
+To be upgraded, a cluster must have each of its nodes individually upgraded in turn; and the upgrade procedure for the cluster must therefore be selected in regard to whether the cluster is required to continue serving data, or to cease serving data, during the cluster-upgrade.
 A review of the factors that determine the appropriateness of an upgrade-procedure is provided in xref:install:upgrade-procedure-selection.adoc[Upgrade Procedure-Selection].
 
 [#supported-upgrade-paths]
 == Upgrade Paths
 
 An upgrade _path_ declares that the upgrade of one version of Couchbase Server to another is _supported_.
 The tables in the following subsections list upgrade paths for Enterprise Edition and for Community Edition, respectively.
-Each instance of the sign -> declares support for the upgrade of the server-version on the left of the sign to the server-version on the right.
-Note that upgrade between non-adjacent version-numbers is frequently _not_ supported: for example, to upgrade from 5.5.0+ to 7.2, upgrade must be performed _twice_; first from 5.5.0+ to 6.6, and secondly from 6.6 to 7.2.
+Each instance of the{nbsp}`->`{nbsp}sign declares support for the upgrade of the server-version on the left of the sign to the server-version on the right.
+
+.Upgrading between non-adjacent version numbers is usually _not_ supported.
+[NOTE]
+====
+For example, to upgrade from *5.5.0+* to *7.2*, then _two_ upgrades must be performed:
+first, from** 5.5.0+** to** 6.6**,
+and then secondly, from *6.6* to *7.2*.
+====
 
 All supported upgrades can be performed with the cluster either _offline_ or _online_.
 
-[IMPORTANT]
-====
+
+
+== How to Upgrade Your Cluster
+
 If you are upgrading several nodes at once, then the version of the software on each node must be kept in step throughout the upgrade process. +
 For example, if you are upgrading three enterprise nodes (`*Node{nbsp}1*`, `*Node{nbsp}2*` and `*Node{nbsp}3*`) from version 5.0x to 7.2, then you would use the following sequence:
 
+.Upgrading from version `5.1x` to `7.2.4`
+====
 [cols="1,2,2"]
 |===
 | Step | Description | Upgrades
 
+
+
 | {counter: upgrade}
-| Upgrade all nodes from 5.0x to 5.1x
+| Upgrade all nodes from 5.1x to 6.0
 |
-`*Node{nbsp}1*` => 5.0x -> 5.1x +
-`*Node{nbsp}2*` => 5.0x -> 5.1x +
-`*Node{nbsp}3*` => 5.0x -> 5.1x
+
+`*Node{nbsp}1*` => 5.1x -> 6.0 +
+`*Node{nbsp}2*` => 5.1x -> 6.0 +
+`*Node{nbsp}3*` => 5.1x -> 6.0
 
 | {counter: upgrade}
-| Upgrade all nodes from 5.1x to 6.6
+| Upgrade all nodes from 6.0 to 6.6
 |
-
-`*Node{nbsp}1*` => 5.1x -> 6.6 +
-`*Node{nbsp}2*` => 5.1x -> 6.6 +
-`*Node{nbsp}3*` => 5.1x -> 6.6
+`*Node{nbsp}1*` => 6.0 -> 6.6 +
+`*Node{nbsp}2*` => 6.0 -> 6.6 +
+`*Node{nbsp}3*` => 6.0 -> 6.6
 
 | {counter: upgrade}
 | Upgrade all nodes from 6.6 to 7.2
 |
-`*Node{nbsp}1*` => 6.6 -> 7.2 +
-`*Node{nbsp}2*` =>  6.6 -> 7.2 +
-`*Node{nbsp}3*` => 6.6 -> 7.2
+`*Node{nbsp}1*` => 6.6 -> 7.2.3 -> 7.2.4 +
```

Comment:
I thought we agreed this is a bad way to represent this? Having two different upgrades in the same Step in this table is very confusing, and would lead to failures if someone followed step 3 node-by-node.